infra/containerregistry.bicep

```diff
@@ -0,0 +1,27 @@
+param location string
+param nameseed string
+param tags object = {}
+
+resource containerRegistry 'Microsoft.ContainerRegistry/registries@2022-02-01-preview' = {
+  name: 'cr${nameseed}${uniqueString(resourceGroup().id, location)}'
```

Comment:
Please start using abbreviations.json, which you can find here: https://github.com/Azure/azure-dev/tree/main/templates/common/infra

Example usage here: https://github.com/Azure/azure-dev/blob/main/templates/common/infra/applicationinsights.bicep

Reply:
Looks like they align to CAF naming, so sure. 🤩